motr/io_req_fop.c

```diff
@@ -150,7 +150,7 @@ static void application_attribute_copy(struct m0_indexvec *rep_ivec,
 	void                   *dst;
 	void                   *src;
 
-	M0_PRE(ioo->ioo_attr.ov_vec.v_nr);
+	M0_PRE(m0__obj_is_di_enabled(ioo));
```

Comment:
It is possible that apart from DI some other info can be passed through attribute, so better not to assert for it.
If ENF_DI is set then proceed, otherwise return from here.

Reply:
done